code/modules/admin/admin_verbs.dm

```diff
@@ -228,19 +228,19 @@ var/list/admin_verbs_mentor = list(
 	set name = "Invisimin"
 	set category = "Admin"
 	set desc = "Toggles ghost-like invisibility (Don't abuse this)"
+
 	if(holder && mob)
 		if(mob.invisibility == INVISIBILITY_OBSERVER)
 			mob.invisibility = initial(mob.invisibility)
-			mob << "\red <b>Invisimin off. Invisibility reset.</b>"
-			mob.icon_state = "ghost"
+			mob << "<span class='danger'>Invisimin off. Invisibility reset.</span>"
+			/*mob.icon_state = "ghost" //this is pointless and ruins an otherwise good feature
 			mob.icon = 'icons/mob/human.dmi'
-			mob.update_icons()
+			mob.update_icons()*/

```

Comment:
It is a nice idea, a visual indicator telling admins they are invisible. This version is just shit.